### PR TITLE
SSCS-10004:Update hearing type for non SSCS1 case

### PR DESCRIPTION
**Pull request by divisathyan**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/SSCS-10004

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitAppealTest.java [hunk 1]
```diff
@@ -124,7 +124,7 @@ private void assertCaseIsExpectedResult(String expectedBody, String expectedStat
 
         assertEquals(expectedState, sscsCaseDetails.getState());
 
-        assertEquals("Joe Bloggs", sscsCaseDetails.getData().getCaseName());
+        //assertEquals("Joe Bloggs", sscsCaseDetails.getData().getCaseName());
```
Comment (jackmaloney) [inline src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitAppealTest.java:127]: Guess take this out before merging

File: src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java [hunk 2]
```diff
@@ -89,7 +89,7 @@ public static SscsCaseData convertSyaToCcdCaseData(SyaCaseWrapper syaCaseWrapper
                             && syaCaseWrapper.getLanguagePreferenceWelsh() != null
                             && syaCaseWrapper.getLanguagePreferenceWelsh()))
                     .ccdCaseId(ccdCaseId)
-                    .caseName(caseName)
+                    //.caseName(caseName)
```
Comment (jackmaloney) [inline src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java:92]: This will need to be removed

**Comment by jackmaloney** [comment]
Whoever ends up merging this, make sure the commented out caseName fields have been removed. These are not needed

File: src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitAppealTest.java (hunk 1 above)
Comment (divisathyan) [reply src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/SubmitAppealTest.java:127]: Yes. Confirmed with Chris that it should be removed but he might merge before this PR
Will remove it in my PR

File: src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java (hunk 2 above)
Comment (divisathyan) [reply src/main/java/uk/gov/hmcts/reform/sscs/transform/deserialize/SubmitYourAppealToCcdCaseDataDeserializer.java:92]: same as above

**Pull request merged**